CHANGELOG.md

```diff
@@ -34,6 +34,10 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 - Deletion of support for the oldest version of the API (v0.0.2) (PR #6844)
 #### Candidate Parameters
 - Consents may now be grouped in UI of consent tab (PR #6042, PR #6044)
+#### API Documentation (**New Module**)
+- Mostly ment for developers, this module provides a user interface to inspect and try LORIS's API.
+- The *superuser* permission is required to access this module.
```

Comment:
I don't think this line belongs in the changelog

Reply:
given the current state of the change log I think we can keep it there and we will go through a major cleanup right after. I think more info is better than less right now

Reply:
It's module documentation, not a change. It doesn't belong in the changelog.

Reply:
done